verilog/formatting/tree_unwrapper.cc

```diff
@@ -837,9 +839,19 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeLeaf& leaf) {
     StartNewUnwrappedLine();
   }
 
-  // Advances NextUnfilteredToken(), and extends CurrentUnwrappedLine().
-  // Should be equivalent to AdvanceNextUnfilteredToken().
-  AddTokenToCurrentUnwrappedLine();
+  if (tag == ',' && CurrentUnwrappedLine().IsEmpty()) {
+    // Partition would begin with a comma,
+    // instead add this token to previous partition
+    AddTokenToCurrentUnwrappedLine();
+    MergeLastTwoPartitions();
+    // Put next token on new partition.
+    // If that token starts a new partition as well, it will just be reused.
+    StartNewUnwrappedLine();
```

Comment:
The logic looks fine.  However, can we group in the switch statement below inside `case ','` (line 861)?  For this leaf-visiting function I prefer to keep the flow that shows distinct phases: pre-token (catch-up), token-handling (just `AddTokenToCurrentUnwrappedLine()` alone), post-token (look-ahead), partition tree adjustments (where the next switch statement is).

Reply:
Done, I moved it to the existing `case`.